test/gutenberg-test-themes/behaviors-enabled/index.php

```diff
@@ -0,0 +1 @@
+<?php // phpcs:disable
```

Comment:
If these (this one and the next below it) are files which should _never_ be included in a scan, you could consider adding an exception to the ruleset instead of a comment.

Reply:
Added an exclusion in the ruleset file 👍